ci_scripts/check_fauxpas.sh

```diff
@@ -20,3 +20,4 @@ fi
 echo "Linting with Faux Pas..."
 
 fauxpas check Stripe.xcodeproj/ --target "StripeiOSStatic" --configFile "./FauxPasConfig/main.fauxpas.json" --minErrorStatusSeverity Concern
```

Comment:
Ah right. I'll just `set -e` on both lines here so future copy pastes dont mess that up also.